src/daemon/devnode.c

```diff
@@ -73,9 +100,11 @@ void _updateconnected(){
     if(!written)
         fputc('\n', cfile);
     fclose(cfile);
-    chmod(cpath, S_GID_READ);
+    
+    (void)check_chmod(cpath, S_GID_READ);
```

Comment:
I wouldn't call this any better than the original code. Casting it to `void` is not error handling.